examples/pylint/w3601_bad_chained_comparison.py

```diff
@@ -0,0 +1,15 @@
+"""Examples for W3601 bad-chained-comparison"""
+
+x = 5
+
+if 0 < x is 5:  # bad-chained-comparison
```

Comment:
This isn't bad, but the convention followed in the rest of the documentation is to leave a comment saying `Error on this line`, since the error type is implied by the filename/documentation section.